main.js

```diff
@@ -96,34 +143,38 @@ function toggleTitleVisibility() {
 // Creates a grid from the inputed column and rows
 function createGridItems() {
     removeItems();
-    let numberOfColumns = columnInput.value;
-    let numberOfRow = rowInput.value;
-    let itemWidth =  window.innerWidth/numberOfColumns;
+    const numberOfColumns = columnInput.value;
+    const numberOfRows = rowInput.value;
+    const itemWidth =  window.innerWidth / numberOfColumns;
 
-    for (i=0; i<(numberOfColumns * numberOfRow); i++) {
-        addItem()
+    let index = 0
+    for (row = 0; row < (numberOfRows); row++) {
+        for (col = 0; col < (numberOfColumns); col++) {
+            addItem();
```

Comment:
It doesn't look like `col` and `row` are being used inside the loop. An alternative to two for loops then could be to multiply `numberOfRows` and `numberOfColumns` and loop through that once.

Reply:
Yes, more old code I forgot about, thank you. Refactored.